beta-src/src/App.tsx

```diff
@@ -12,11 +12,15 @@ const App: React.FC = function (): React.ReactElement {
   const urlParams = new URLSearchParams(window.location.search);
   const currentGameID = urlParams.get("gameID");
   const dispatch = useAppDispatch();
+  const countryID = useAppSelector((state) =>
+    String(state.game.overview.user.member.countryID),
+  );
   dispatch(fetchGameOverview({ gameID: currentGameID as string }));
   dispatch(
     fetchGameMessages({
       gameID: currentGameID as string,
-      countryID: "0" as string,
+      countryID,
```

Comment:
can we just do countryID as string here to keep it consistent with gameID? I think it is preferable than type casting it as we're fetching it from global state.

Reply:
@bryancodazen I initially tried that but got a type error 
<img width="1125" alt="Screen Shot 2022-05-09 at 9 26 47 PM" src="https://user-images.githubusercontent.com/1848053/167542804-4ca77633-97f7-42ab-9e2d-c7e697f98cd1.png">

I would have to do the following.
`countryID: countryID as unknown as string,`